dagshub/streaming/filesystem.py

```diff
@@ -124,12 +124,15 @@ def __init__(self,
         else:
             # Check Git credential stores
             proc = subprocess.run(['git', 'credential', 'fill'],
-                                input=f'url={repo_url}'.encode(),
-                                capture_output=True)
+                                  input=f'url={repo_url}'.encode(),
+                                  capture_output=True)
             answer = {line[:line.index('=')]: line[line.index('=')+1:]
-                        for line in proc.stdout.decode().splitlines()}
+                      for line in proc.stdout.decode().splitlines()}
             if 'username' in answer and 'password' in answer:
                 self.auth = (answer['username'], answer['password'])
+            response = self._api_listdir('')
```

Comment:
I tried to see if private repos work now, and I'm crashing here now.
Seems like the result of this is cached after line 120 and it doesn't retry to access the folder again